ang/civicase/case/details/summary-tab/case-details-summary-next-non-milestone-activity.html

```diff
@@ -0,0 +1,47 @@
+<div class="panel civicase__summary-tab-tile civicase__panel-transparent-header civicase__summary-tab-tile--responsive">
+  <div ng-if="isMainContentVisible()" class="panel-heading">
+    <h3 class="panel-title">Next Activity</h3>
+  </div>
+  <div ng-if="isMainContentVisible()" class="panel-body">
+    <div
+      case-activity-card="item.nextActivityNotMilestone"
+      crm-popup-form-success="pushCaseData($data.civicase_reload[0])"
+      refresh-callback="refresh"
+      mode="big"
+      type="'activity'"
+      case="item">
+    </div>
+  </div>
+
+  <!-- Placeholder -->
+  <div ng-if="isPlaceHolderVisible()" class="civicase__loading-placeholder__oneline" style="width: 10em; margin-bottom: 10px;"></div>
```

Comment:
on the one hand I'm happy that the HTML parts are stored into easy to find folders, but on the other I find having the actual content and the placeholder content in a single file hard to follow.

Is it possible to have something like this for everyone:

```html
<div class="panel civicase__summary-tab-tile civicase__panel-transparent-header civicase__summary-tab-tile--responsive">
  <div ng-if="isMainContentVisible()" class="panel-heading">
    <h3 class="panel-title">Next Activity</h3>
  </div>
  <div ng-if="isMainContentVisible()" class="panel-body">
    <div
      case-activity-card="item.nextActivityNotMilestone"
      crm-popup-form-success="pushCaseData($data.civicase_reload[0])"
      refresh-callback="refresh"
      mode="big"
      type="'activity'"
      case="item">
    </div>
  </div>

  <div
    ng-if="isPlaceHolderVisible()"
    ng-include="'some-path-next-to-this-file/name-of-file__placeholder.html'">
  </div>
</div>
```

Reply:
@reneolivo I initially tried this, but that way a new html element gets created because of `ng-include` and it changes the html tree, which messes up with the styles. Now I have created `civicaseIncludeReplace` to fix this particular issue, but still I was a bit reluctant to use this all over the place, because there are so many file changes in this PR, that in case something goes wrong, it will take much more time to everything.
Can we keep it as it is for now? as we have a good start, and we can build on top of this in future, where we will change few files at a time, so that its easier to fix in case of regression.
Thats why I added the `<!-- placeholder -->` comment in each file so that its easier to understand.

Reply:
Agreed. Let's try to move them as we touch the files. This is way better than the previous implementation anyways.